wp-content/themes/reconasia/assets/_scss/layout/_footer.scss

```diff
@@ -60,31 +72,36 @@
   &__logo {
     grid-area: logo;
     max-width: 300px;
+  }
 
-    &,
-    &:visited {
-      // color: $color-text-gray-10;
-    }
+  &__desc {
+    grid-area: csisdesc;
   }
 
   .widget:first-of-type {
-    grid-area: csisdesc;
+    margin-bottom: rem(24);
   }
 
   .widget:nth-of-type(2) {
-    grid-area: magdesc;
+    margin-bottom: rem(32);
+  }
```

Comment:
This is effecting the second paragraph in the description and the newsletter section, which don't share a common `margin-bottom`. You'll to use more specific selectors to add the correct margin to both of these elements.